packages/aws-cdk-lib/aws-ecs/lib/log-drivers/aws-log-driver.ts

```diff
@@ -135,7 +136,16 @@ export class AwsLogDriver extends LogDriver {
       ? `${this.props.maxBufferSize.toBytes({ rounding: SizeRoundingBehavior.FLOOR })}b`
       : undefined;
 
-    this.logGroup.grantWrite(containerDefinition.taskDefinition.obtainExecutionRole());
+    // These policies are required for the Execution role to use awslogs driver.
+    // In cases where `addToExecutionRolePolicy` is not implemented in some cases,
+    // for example, when used from aws-batch construct,
```

Comment:
for example,
https://github.com/aws/aws-cdk/blob/607dccb0fd920d25f0fe2613b83c9830322c439e/packages/aws-cdk-lib/aws-batch/lib/ecs-container-definition.ts#L610